### DISCUSSION TITLE
MPRLS or MPR pressure sensor support

### DISCUSSION THREAD

**Discussion by skupu** [arendst/Tasmota#19300] [Q&A]
Dear Theo @arendst,

I need to measure pressure difference - the most suitable way would be with one DFRobot SEN0343 or two of SparkFun SEN-16476 or Adafruit 3965.

Is there any chance to make use of those with Tasmota?

There are libraries:
https://github.com/DFRobot/DFRobot_LWLP
https://github.com/sparkfun/SparkFun_MicroPressure_Arduino_Library
https://github.com/adafruit/Adafruit_MPRLS

but honestly I don't know how to integrate them into own compiled tasmota, if possible.

Could you please support?

Thank you!

**Comment by skupu** [comment]
Anyone?

**Reply by barbudor** [reply]
Posting twice the same request won't help
At worst, you will piss off the team and get ghosted

To create a driver start reading this : https://tasmota.github.io/docs/For-Developers/
Then try to identify the most similar existing driver and get inspiration from it

**Reply by skupu** [reply]
So, as I couldn't find a solution myself and noone offered to help, I had to use Arduino.
It is not what I have been expecting, but it works.

Still trying to make it work with tasmota.
If anyone could help...
